docs/configure-artifact-repository.md

```diff
@@ -410,30 +410,32 @@ In addition to managed identities, Argo workflows also support authentication us
 
 ### Using Azure Shared Access Signatures (SAS)
 
-If you do not wish to use an access key, you may also use a [shared access signature (SAS)](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json).
+> v3.6 and after
```

Comment:
Thanks for the review comments @agilgur5 , I'll fix them as soon as I can.

Meanwhile, would it be possible to include the Azure SAS token support feature in an earlier 3.5.x patch version rather than the next major version(3.6)?

Reply:
We normally do not backport features (unless they fix something broken)

Reply:
Thank @agilgur5 for your prompt responses!

Could the SAS support be considered as a fix and included in the next patch since Azure SAS tokens break our existing adapter instance and the fix handles them separately via a different adapter instance. 

Do we also have a timeline of when the next patch and v3.6 is expected to be released?

Reply:
The issue was written as a feature request and the PR as a feature. Those certainly don't sound like bugfixes to me.

If we were to backport features into patch releases, then patch releases would have significant changes that may be disruptive. You can't have it both ways. Something that benefits you is not necessarily in the best interest of the project. It also sets poor precedent.

3.5.x is unstable as-is due to regressions from features. You asked about the 3.6 timeline, and it has been [delayed while 3.5.x is unstable](https://cloud-native.slack.com/archives/C01QW9QSSSK/p1721095606777589?thread_ts=1720689182.756049&cid=C01QW9QSSSK). I'll have the next 3.5.x patch out this week (I was working on it last night)

I also don't see anything particularly special about this feature that warrants backporting and it was not very popular either (3 upvotes).

You are also more than welcome to make your own custom image if you want a feature quicker. This is OSS after all, you have the freedom to use the code in different manners and are not constrained by maintainers.

Reply:
Thanks @agilgur5, that's a fair point and I understand the reasoning!
We'll try and keep an eye out for `3.6`.